tools/ci/dm.sh

```diff
@@ -38,12 +38,6 @@ do
 		sed -i '1s/^/#define '$arg'\n/' $dmepath.mdme
 		continue
 	fi
-	if [[ $var == -M* ]]
-	then
-		sed -i '1s/^/#define MAP_OVERRIDE\n/' $dmepath.mdme
-		sed -i 's!// BEGIN_INCLUDE!// BEGIN_INCLUDE\n#include "_maps\\'$arg'.dm"!' $dmepath.mdme
-		continue
-	fi
```

Comment:
@Kyep pinging to make sure this doesn't break backend tooling, as the `-m` arg is surplus at this point, and entirely unsupported in the code now.

Reply:
Currently, the backend does this:
tools\ci\dm.sh paradise.dme -Mcyberiad
I assume this is no longer supported, and after this PR is merged I need to delete the "-Mcyberiad" from this line.

Reply:
Yes that is the case, however `/datum/map/cyberiad` is the default map datum
https://github.com/ParadiseSS13/Paradise/blob/8e4f17832f6fdc51c72ebd461438b27711040cad/code/controllers/subsystem/mapping.dm#L25

Reply:
So what you're telling me is that I can delete the -Mcyberiad right now and it won't matter because its already the default.

Reply:
Yep. If you dont specify a map it builds cyberiad by default
```
$ tools/ci/dm.sh paradise.dme
DM compiler version 513.1536
loading paradise.mdme
loading interface\skin.dmf
loading map_files\generic\centcomm.dmm
loading map_files\cyberiad\cyberiad.dmm
loading map_files\generic\Lavaland.dmm
saving paradise.mdme.dmb (DEBUG mode)
paradise.mdme.dmb - 0 errors, 0 warnings (1/28/21 9:07 pm)
Total time: 0:31
paradise.mdme.dmb - 0 errors, 0 warnings (1/28/21 9:07 pm)
```